src/diagnostics/radiation_diagnostics.jl

```diff
@@ -857,3 +857,88 @@ add_diagnostic_variable!(
     comments = "Upwelling clear-sky longwave radiation at the top of the atmosphere",
     compute! = compute_rlutcs!,
 )
+
+
+###
+# Effective radius for liquid clouds (3d)
+###
+compute_reffclw!(out, state, cache, time) =
+    compute_reffclw!(out, state, cache, time, cache.atmos.radiation_mode)
+compute_reffclw!(_, _, _, _, radiation_mode::T) where {T} =
+    error_diagnostic_variable("reffclw", radiation_mode)
+
+function compute_reffclw!(
+    out,
+    state,
+    cache,
+    time,
+    radiation_mode::Union{
+        RRTMGPI.AllSkyRadiationWithClearSkyDiagnostics,
+        RRTMGPI.AllSkyRadiation,
+    },
+)
+    FT = eltype(state)
+    if isnothing(out)
+        return Fields.array2field(
+            cache.radiation.rrtmgp_model.center_cloud_liquid_effective_radius,
+            axes(state.c),
+        ) .* FT(1e-6) # RRTMGP stores r_eff in microns
+    else
+        out .=
+            Fields.array2field(
+                cache.radiation.rrtmgp_model.center_cloud_liquid_effective_radius,
+                axes(state.c),
+            ) .* FT(1e-6)
+    end
+end
+
+add_diagnostic_variable!(
+    short_name = "reffclw",
+    long_name = "Effective radius for liquid clouds",
+    standard_name = "effective_radius_of_cloud_liquid_particles",
+    units = "m",
+    comments = "In-cloud ratio of the third moment over the second moment of the particle size distribution. Set to zero outside of clouds.",
```

Comment:
If you do the ifelse thing in the end, let's add in the comments that it is set to zero when there is no cloud, so users know the monthly mean diagnostics is inaccurate.

Reply:
I have it now in the comments. Would you prefer in the long name?

Reply:
No this is good enough, thanks!